### PR TITLE
Install tfenv on build image

### PR DESCRIPTION
**Pull request by defond0**
This will install tfenv on our build image as well as a few choice versions of terraform.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -22,18 +23,21 @@ RUN apk --no-cache --update add \
...
-    wget https://releases.hashicorp.com/terraform/0.11.14/terraform_0.11.14_linux_amd64.zip -O tmp.zip && unzip tmp.zip -d /usr/local/bin/; rm tmp.zip \
-      && \
-    rm -rf .terraform \
-      && \
-    wget https://github.com/go-swagger/go-swagger/releases/download/v0.19.0/swagger_linux_amd64 -O /usr/local/bin/swagger \
-      && \
+    wget https://github.com/go-swagger/go-swagger/releases/download/v0.19.0/swagger_linux_amd64 -O /usr/local/bin/swagger && \
     chmod +x /usr/local/bin/swagger
 
+RUN git clone https://github.com/tfutils/tfenv.git ${HOME}/.tfenv && \
```
Comment (myrcutio) [inline Dockerfile]: Is it necessary to download the whole repo if we just need two binaries?

Could do a wget on the two most up-to-date paths: 
https://raw.githubusercontent.com/tfutils/tfenv/master/bin/tfenv
https://raw.githubusercontent.com/tfutils/tfenv/master/bin/terraform

File: Dockerfile [hunk 2]
```diff
@@ -22,18 +23,21 @@ RUN apk --no-cache --update add \
...
+    tfenv install 0.12.12 
+
 RUN wget http://central.maven.org/maven2/io/swagger/swagger-codegen-cli/2.4.8/swagger-codegen-cli-2.4.8.jar -O /usr/local/bin/swagger-codegen.jar \
       && \
     chmod +x /usr/local/bin/swagger-codegen.jar
 
 RUN echo 'exec java -jar /usr/local/bin/swagger-codegen.jar "$@"' >> /usr/local/bin/swagger-codegen \
       && \
-    chmod +x /usr/local/bin/swagger-codegen
+    chmod +x /usr/local/bin/swagger-codegen
```
Comment (myrcutio) [inline Dockerfile]: pretty sure we don't need swagger-codegen anymore, since the web side uses the npm install instead

File: Dockerfile (hunk 1 above)
Comment (defond0) [reply Dockerfile]: makes good sense

File: Dockerfile [hunk 3]
```diff
@@ -4,6 +4,7 @@ RUN apk --no-cache --update add \
         bash \
         build-base \
         gcc \
+        curl \
```
[the remark this answers is not in the document]
Comment (defond0) [reply Dockerfile]: tfenv uses curl to install it's versions.

**Pull request merged**